OpenEdXMobile/build.gradle

```diff
@@ -170,6 +170,7 @@ dependencies {
 
     // Google Firebase
```

Comment:
Note: We need to update this version soon.  I will look at adding a story to the techdebt back log.

Reply:
Its going to be done in this PR.
https://github.com/edx/edx-app-android/pull/1125